connect-wallet-nextjs/GENERATED-README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
I kept that file to avoid confusion on the Next.js started kit but we can delete the file altogether here.